wallet/build.gradle.kts

```diff
@@ -27,29 +23,14 @@ kotlin {
     compilerOptions {
         allWarningsAsErrors = true
     }
-
-    androidTarget {
-        @OptIn(ExperimentalKotlinGradlePluginApi::class)
-        compilerOptions {
-            jvmTarget.set(JvmTarget.JVM_17)
-        }
-    }
-
-    sourceSets {
-        val androidMain by getting {
-            dependencies {
-                implementation(project(":multipaz"))
-            }
-        }
-    }
 }
 
 android {
-    namespace = "org.multipaz_credential.wallet"
+    namespace = "org.multipaz.wallet"
     compileSdk = libs.versions.android.compileSdk.get().toInt()
 
     defaultConfig {
-        applicationId = "org.multipaz_credential.wallet"
+        applicationId = "org.multipaz.wallet"
```

Comment:
can we change this to **org.multipaz.androidWallet** or similar? I'd like to use **org.multipaz.wallet** for our multiplatform rewrite...

Reply:
Changed to org.multipaz.androidwallet.